extensions/machine-learning-services/src/widgets/configTable.ts

```diff
@@ -0,0 +1,182 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { ApiWrapper } from '../common/apiWrapper';
+import { ServerConfigManager } from '../serverConfig/serverConfigManager';
+import * as constants from '../common/constants';
+
+export class ConfigTable {
+	private _statusTable: azdata.DeclarativeTableComponent;
+
+	constructor(private _apiWrapper: ApiWrapper, private _serverConfigManager: ServerConfigManager, private _modelBuilder: azdata.ModelBuilder, private _loadingComponent: azdata.LoadingComponent) {
+		this._statusTable = this._modelBuilder.declarativeTable()
+			.withProperties<azdata.DeclarativeTableProperties>(
+				{
+					columns: [
+						{ // Config
+							displayName: constants.mlsConfigTitle,
+							valueType: azdata.DeclarativeDataType.component,
+							isReadOnly: true,
+							width: 175,
+							headerCssStyles: {
+								'border': 'none',
+								'background-color': '#FFFFFF',
+								...constants.cssStyles.tableHeader
+							},
+							rowCssStyles: {
+								'border-top': 'solid 1px #ccc',
+								'border-bottom': 'solid 1px #ccc',
+								'border-left': 'none',
+								'border-right': 'none'
+							},
+						},
+						{ // Status icon
+							displayName: constants.mlsConfigStatus,
+							ariaLabel: constants.mlsConfigStatus,
+							valueType: azdata.DeclarativeDataType.component,
+							isReadOnly: true,
+							width: 25,
+							headerCssStyles: {
```

Comment:
>headerCssStyles [](start = 7, length = 15)

This is missing the ...constants.cssStyles.tableHeader styles so it doesn't look like the CONFIG title. You should make them consistent